CONTRIBUTING.md

```diff
@@ -78,33 +78,44 @@ To set up `pysatNASA` for local development:
 
 
 4. When you're done making changes, run all the checks to ensure that nothing
-   is broken on your local system, as well as check for flake8 compliance:
+  is broken on your local system:
 
-   ```
-    pytest -vs --flake8 pysatNASA
-   ```
+  ```
+  pytest PACKAGENAME
```

Comment:
```suggestion
  pytest pysatNASA
```

Could add the flags as well, if you like.

Reply:
Updated below with improved flake8 tests.  Added to draft pull at ecosystem site